Targets/ICS/SiemensTIA.tkape

```diff
@@ -0,0 +1,16 @@
+Description: Copy Siemens TIA Settings
+Author: Olaf Schwarz (@b00010111)
+Version: 1.0
+Id: 0e43ebe5-cd5b-4487-ab41-66f46ebfc5c9
+RecreateDirectories: true
+Targets:
+    -
+        Name: Siemens TIA Settings
+        Category: ICS
+        Path: C:\Users\%user%\AppData\Roaming\Siemens\Automation\Portal*\Settings\
+        Recursive: true
+
+# Documentation
+# The Settings.xml file contains information about opened project files, last storge location used and so on.
+# You can find more info on the Settings.xml file in part 2 of this blog post series: https://blog.nviso.eu/series/investigating-an-engineering-workstation/  
+# There is a tool available to parse the Settings.xml file: https://github.com/00010111/parse_tSettings The verbose output also provides information on the content of the Settings.xml file

```

Comment:
Are we going to have a Module made for this tool?

Reply:
If I find the time for yes. If someone else wants to give it a go, this is also fine with me.